xen/include/public/io/displif.h

```diff
@@ -0,0 +1,620 @@
+/******************************************************************************
+ * displif.h
+ *
+ * Unified display device I/O interface for Xen guest OSes.
```

Comment:
Well, started to describe and it comes out that the protocol is mostly self-explanatory.
Focused more on TODO part to better demonstrate what this protocol can be